projects/js-packages/charts/src/stories/sample-data/index.ts

```diff
@@ -0,0 +1,712 @@
+		previousValue: 580000,
+		currentShare: 50,
+		previousShare: 46,
+		delta: 7.8,
+	},
+];
+
+export const leaderboardNegativeGrowth = [
+	{
+		id: 'negative1',
+		label: 'Declining Channel',
+		currentValue: 5000,
+		previousValue: 8000,
+		currentShare: 62.5,
+		previousShare: 100,
+		delta: -37.5,
+	},
+	{
+		id: 'negative2',
+		label: 'Another Declining',
+		currentValue: 3000,
+		previousValue: 6000,
+		currentShare: 37.5,
+		previousShare: 75,
+		delta: -50,
+	},
+	{
+		id: 'negative3',
+		label: 'Slight Decline',
+		currentValue: 4500,
+		previousValue: 4800,
+		currentShare: 56.25,
+		previousShare: 60,
+		delta: -6.25,
+	},
+];
+
+export const leaderboardWithImageColor = [
+	{
+		id: 'direct',
+		label: 'Direct',
+		currentValue: 12500,
+		previousValue: 10000,
+		currentShare: 100,
+		previousShare: 80,
+		delta: 25,
+		imageColor: '#007aff',
+	},
+	{
+		id: 'social',
+		label: 'Social Media',
+		currentValue: 8750,
+		previousValue: 9500,
+		currentShare: 30,
+		previousShare: 76,
+		delta: -7.9,
+		imageColor: '#ffc0cb',
+	},
+	{
+		id: 'referral',
+		label: 'Referral',
+		currentValue: 8750,
+		previousValue: 9500,
+		currentShare: 10,
+		previousShare: 16,
+		delta: -7.9,
+		imageColor: '#00ff00',
+	},
+];
+
+/**
+ * Conversion funnel sample data
+ */
+export const sampleFunnelData = [
+	{
+		id: 'sessions',
+		label: 'Sessions',
+		rate: 100,
+		count: 10000,
+	},
+	{
+		id: 'cart',
+		label: 'Cart',
+		rate: 71.1,
+		count: 7110,
+	},
+	{
+		id: 'checkout',
+		label: 'Checkout',
+		rate: 52.5,
+		count: 5250,
+	},
+	{
+		id: 'purchase',
+		label: 'Purchase',
+		rate: 10.3,
+		count: 1030,
+	},
+];
+
+export const lowConversionData = [
+	{
+		id: 'sessions',
+		label: 'Sessions',
+		rate: 100,
+		count: 5000,
+	},
+	{
+		id: 'cart',
+		label: 'Cart',
+		rate: 45.2,
+		count: 2260,
+	},
+	{
+		id: 'checkout',
+		label: 'Checkout',
+		rate: 28.8,
+		count: 1440,
+	},
+	{
+		id: 'purchase',
+		label: 'Purchase',
+		rate: 6.4,
+		count: 320,
+	},
+];
+
+export const highConversionData = [
+	{
+		id: 'sessions',
+		label: 'Sessions',
+		rate: 100,
+		count: 8000,
+	},
+	{
+		id: 'cart',
+		label: 'Cart',
+		rate: 85.3,
+		count: 6824,
+	},
+	{
+		id: 'checkout',
+		label: 'Checkout',
+		rate: 72.1,
+		count: 5768,
+	},
+	{
+		id: 'purchase',
+		label: 'Purchase',
+		rate: 18.7,
+		count: 1496,
+	},
+];
+
+/**
+ * Bar list chart sample data
+ */
+export const barListSample = [
+	{
+		group: 'primary',
+		label: 'Jan 21-Aug 8, 2024',
+		data: [
+			{ label: 'Organic search', value: 30000 },
+			{ label: 'Affiliates', value: 19000 },
+			{ label: 'Display', value: 18000 },
+			{ label: 'Organic shopping', value: 16000 },
+		],
+	},
+	{
+		group: 'comparison',
+		label: 'Jan 21-Aug 8, 2023',
+		data: [
+			{ label: 'Organic search', value: 20000 },
+			{ label: 'Affiliates', value: 15000 },
+			{ label: 'Display', value: 19900 },
+			{ label: 'Organic shopping', value: 20500 },
+		],
+	},
+];
+
+export const salesByProduct = [
```

Comment:
We've got a mixture of naming conventions in this file; some like `barListSample` and `leaderboardSample` which reflect their intended usage, some like `salesByProduct` which describe the type of data, and some like `olympicMedals` which describe what the data represents. I think we should standardise, and I'm leaning towards the `salesByProduct` version, as that's most closely tied to what our charts present. What do you think?